src/sections/footer.html

```diff
@@ -1,4 +1,30 @@
-<div>
-    <h1>Footer</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
-</div>
\ No newline at end of file
+<!-- Parent -->
+<div class="bg-webGrey px-5 lg:px-9 pt-20 pb-9">
+  <!-- Image   -->
+  <div class="flex justify-center w-full">
+    <img src="/images/red-shape.svg" alt="" class="md:w-[50%] w-[100%]" />
```

Comment:
Avoid empty attributes and using arbitrary values when working with tailwind's utility classes because they can have some effects such as hard to read classes and mostly also affect responsiveness. so instead of using arbitrary values you can use their corresponding tailwind classes.
```suggestion
    <img src="/images/red-shape.svg" alt="eg:sharp shape" class="md:w-1/2 w-full" />
```